pkg/importer/data-processor.go

```diff
@@ -393,3 +392,12 @@ func (dp *DataProcessor) calculateTargetSize() int64 {
 func (dp *DataProcessor) PreallocationApplied() string {
 	return dp.preallocationApplied
 }
+
+func (dp *DataProcessor) getUsableSpace() int64 {
+	blockSize := int64(512)
```

Comment:
Shouldn't the block size be a constant variable or dynamically defined?
I think that dynamically is better since It might not be always 512. 
It seems that the function blockSize, err := GetAvailableSpaceBlock(fileName) could be used here

Reply:
My motivation or the 512 hard-coded choice was seeing a lot of code inside qemu/qemu-img.c doing
DIV_ROUND_UP(..., BDRV_SECTOR_SIZE)
where BDRV_SECTOR_SIZE is 512.